### DISCUSSION TITLE
Customizable 'Navigation Bar'

### DISCUSSION THREAD

**Discussion by Eosphorus** [xiaoyifang/goldendict-ng#171] [Ideas]
Hi there,

I'd like to propose a little extension of the 'Navigation Bar', that is to say four more buttons. There are functions I personally use constantly and I find it a tad cumbersome to access them via menu or keyboard shortcuts. That's fine alright but a little mouse click on a button would be swifter.

I am aware that there is/was some wiggle room here by customizing the stylesheet file "qt-style.css". Unfortunately, I can't find any documentation any more. A hint?

And then there was never full access to all or at least the basic menu functions. Back in the day I managed to adjust a few things here but it seems that with the switch to "QtWebEngine" or even earlier things have changed.

To be specific, I miss:
	- 'Search in page'
	- 'Full-text search'
	- 'Dictionaries...'
	- and in the Dicionaries dialog box 'Dictionaries/Files': 'Rescan now'

Generally, a freely customizable 'Navigation Bar' would be highly appreciated.

Thanks for reading.

**Comment by xiaoyifang** [comment]
currently you can use shortcut to access them.
- 'Search in page'               Ctrl+F
- 'Full-text search'               Ctrl+Shift+F
- 'Dictionaries...'              F3
